### PR TITLE
[READY]Armadyne Anti Trust Act Part 2: Return of The Ballistic Security

### PR DESCRIPTION
**Pull request by projectkepler-ru**
## About The Pull Request
This massively reshuffle the gun cargo and security focus, instead focusing on making sure energy weapon is more readily available, while upping the cost of ballistic weapory, making them rarer and more expensive ish
However, in turn, as ballistic sec is pushed toward becoming a research tech, this allow there to be more engaging gameplay with balistic, because,  you as antag will be able to know ballistic is being focused as a tech, which gives you better fore-warning

while at the same time, on sec side,  you don't have that same ease of ballistic availability

Tl;dr
ballistic are more material intensive, can't be as easily purchased, require research to be particularly useful
more intended for long game
energy weapon are cheaper to get, can be more easily purchased, does not require as much research to be useful
still affected by emp

m45a5 sprite from #3595

lots of gun sprite from #2860

all other sprite are already in skyrat, are made by me, the IDH Project(@Wolf-751 ,@moostard)  or Kepler Station SS13 and other benefactors of our project

Gun vendors that have their import commented out are mainly for archival purpose so we can make sure any port from nova or bubber is maintained and we know what the original stuff is supposed to be
the RCF and prybar are removed from import but colonization crate still exist

## How This Contributes To The Skyrat Roleplay Experience
Having less antag dies horribly in 1-2 hours of the round and balistic sec being something they can tangibly keep track off of if they are paying attention to science or paying attention to radio massively improve the experience, vs the silent cargo buying a ton of guns, sec getting even just one make it nearly unplayable

Details later
## Proof of Testing
image

image
image

<details>
<summary>Screenshots/Videos</summary>
  
image

</details>

## Changelog
:cl: EspeciallyStrange, Waterpig & Wolf751
add: Sporting Rifle in .40
add: Printable 10mm Speedloader, they're quite cheap but not at all that great.
add: Round start security pistol set in 9mm, 10mm revolver and also an alert locked flechette rifle
add: Conversion kit for guns, alongside the blueshield/NTC getting a new magistrate weapon beacn, letting them call in the energy revolver, the HoS revolver from bubberstation and the M45A5 from ye olden time
add: Research for the new technode, military grade weaponry 1, 2 , 3 With the last one requiring you to have illegal tech
balance: Due to the on-going Sol Conflict not reflected in  C-3 Sector. skild, takbok, bogseo, AMR, LMG are no longer obtainable, cargo took a very heavy hit due to embargo. A certain individual with a sequencer might be able to reach this however.
add: a new helmet type that is inoffensive to the tactical group of the playerbase, people wanted it so we'll have it, at the same time, you can now reskin the 'classic' helmet' to have an alternate blue variant too
balance: made laser much easier to get while ballistic are abit more rare and more expensive
balance: massively overhauled how ballistic security work, being they are now a mid-late game researchable tech
balance: 90% of ammo are no longer purchasable, instead they are available as tech, see above.
balance; .40 damage has their wound decreased
balance: 9mm damage is reduced to 25, 10mm damage is reduced to 30
balance: laser are 25 damage, as they are supposed to
balance: .38 are now 30 damage
balance: The new C-3 Sector peacekeeper combat shotgun, they're bulky, replacing the old combat shotgun. But they don't fire as fast due to a certain recoil compensation mechanism, however, the HoS shotgun has been given a gyroscopic stabilizer allowing them to be fired one handed
balance: stardust ammo now fire one projectile with 100% embed chance, 5 second removal time instead of 10. and is slightly harder to get
balance: the RCF and prybar are removed from import but colonization crate still exist, this is done to compensate for the fact they are way too easily obtained, requiring no research tech and even provide ore satchel of holding before research for it is acquired. Prybar is only ever used by antag and greytider to get into place, otherwise, if a paramedic need to, they can bug engineering for a prybar when a crate is made

image: added new sprites for the gun, pictured above.
/:cl:

### CODE REVIEW

**Comment by projectkepler-ru** [comment]
This will be finished by 7/7

**Comment by projectkepler-ru** [comment]
@SpaceLoveSs13  I don't have the screenshot up but it's ready now

File: modular_skyrat/modules/company_imports/code/armament_datums/akh_frontier.dm [hunk 1]
```diff
@@ -10,11 +10,11 @@
 
 /datum/armament_entry/company_import/akh_frontier/basic/omni_drill
 	item_type = /obj/item/screwdriver/omni_drill
-
+/*
```
Comment (SpaceLoveSs13) [inline modular_skyrat/modules/company_imports/code/armament_datums/akh_frontier.dm]: Why was this done? You havent documented it.

File: modular_skyrat/modules/company_imports/code/armament_datums/akh_frontier.dm [hunk 2]
```diff
@@ -25,12 +25,12 @@
 
 /datum/armament_entry/company_import/akh_frontier/deployables_fab
 	subcategory = "Deployable Fabrication Equipment"
-
+/*
```
Comment (SpaceLoveSs13) [inline modular_skyrat/modules/company_imports/code/armament_datums/akh_frontier.dm]: This as well

**Review by SpaceLoveSs13 — DISMISSED** [review]

**Comment by projectkepler-ru** [comment]
No further update. this is ready

**Comment by projectkepler-ru** [comment]
actually hold on i need to finish up something, it'll be ready proper by sat

**Comment by projectkepler-ru** [comment]
barring any further issues, this is final

**Comment by projectkepler-ru** [comment]
sprite broke

**Pull request merged**